kafka-inmemory/resources/zookeeper.yaml

```diff
@@ -3,8 +3,8 @@ kind: StatefulSet
 metadata:
   name: zookeeper
 spec:
-  serviceName: "zookeeper"
-  replicas: 1
+  serviceName: "zookeeper-headless"
+  replicas: 3
```

Comment:
Since the inmemory deployment is intended to server test/development use case, wouldn't it be better to leave this as 1?

Reply:
Fixed.